WordPress/src/main/java/org/wordpress/android/ui/reader/views/compose/tagsfeed/ReaderTagsFeedPostListItem.kt

```diff
@@ -29,20 +31,27 @@ import androidx.compose.runtime.remember
 import androidx.compose.ui.Alignment
 import androidx.compose.ui.Modifier
 import androidx.compose.ui.draw.clip
+import androidx.compose.ui.graphics.Color
 import androidx.compose.ui.layout.ContentScale
 import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.platform.LocalDensity
 import androidx.compose.ui.res.painterResource
 import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.rememberTextMeasurer
 import androidx.compose.ui.text.style.TextOverflow
 import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.Constraints
 import androidx.compose.ui.unit.dp
 import coil.compose.AsyncImage
 import coil.request.ImageRequest
 import org.wordpress.android.R
-import org.wordpress.android.ui.compose.modifiers.conditionalThen
 import org.wordpress.android.ui.compose.theme.AppColor
 import org.wordpress.android.ui.compose.theme.AppTheme
 import org.wordpress.android.ui.compose.unit.Margin
+import kotlin.math.min
+
+private const val CONTENT_TOTAL_LINES_WITH_INTERACTIONS = 3
+private const val CONTENT_TOTAL_LINES_NO_INTERACTIONS = 4
```

Comment:
Good point @develric! I just tested it with these values and it looks more balanced though I needed to do some tweaking in some spacing and positioning to make sure all cases look good.

The main change I needed to do was:
- Title + excerpt now try to fill all the height available
- Excerpt is centered vertically **when it has the maximum number of lines expected**
- Excerpt is aligned to the top, with an `8dp` top margin **when it has fewer lines than the expected number of lines**

This also makes sure the feature image position is more consistent. Before, the image was aligned to the title + excerpt component, so if it was small, the image would be higher up on the card.

Here are some examples:
Title with 2 lines, no likes/comments:
![image](https://github.com/wordpress-mobile/WordPress-Android/assets/5091503/b51d0934-6ef1-4e68-b0fc-ebec5c6ed2e1)

Title with 2 lines, with likes/comments:
![image](https://github.com/wordpress-mobile/WordPress-Android/assets/5091503/b08dbce3-65f4-4d28-9183-305e32b6c624)

Title with 1 line, no likes/comments:
![image](https://github.com/wordpress-mobile/WordPress-Android/assets/5091503/e8c3e747-e5ed-4970-99d9-0d44f62b14cc)

Title with 1 lines, with likes/comments:
![image](https://github.com/wordpress-mobile/WordPress-Android/assets/5091503/010bfa56-f36d-476b-89e2-2799bbc9e4ec)

---

Another thing I noticed is that since we are always fetching the most recent posts for each tag, **most posts don't have likes and **comments**. And since refreshing the page will almost always (depending on the tag of course) bring new posts, this is always true.

Note that I have 7 followed tags which are pretty generic, so they have posts published with them very frequently, and it's actually hard finding posts with likes/comments.

That said, is the like/comment information useful in this feed? I feel it is making the spacing/position logic for the other elements a bit more complex while being hidden most of the time.

Reply:
@thomashorta thanks for all those thoughts. Taking the discussion about number of lines of title and excerpt to Slack to quickly resolve. I actually want it to be more restrictive, taking up a pretty defined area lining up with the image. Couple others:

- in my design I also thought that we might have too many items between username, title, excerpt, metadata, actions. And I had asked if anyone thought there was one we could drop. Given you've made a case to drop the metadata, I am open to it for sure.
- You did give me a feature idea. Obviously separate to this effort. But what if in the default tags feed those were most liked posts or posts with high likes ratio from the last 1-7 days (depending how often/recent the user visits) and then the full chronological stream is visible when the user taps to filter by tag, or to see all the posts from that tag? It might give a better taste of good content here.

Reply:
> But what if in the default tags feed those were most liked posts or posts with high likes ratio from the last 1-7 days (depending how often/recent the user visits) and then the full chronological stream is visible when the user taps to filter by tag, or to see all the posts from that tag? It might give a better taste of good content here.

That would be interesting. We just need to make sure the content doesn't get stale, so refreshing the screen/coming to it every day actually loads new content.

Regarding the text lines question, as per our discussion on Slack (p1715708170826399/1715699905.724819-slack-C05N140C8H5), I will go with this for now:
- 3 lines for title + excerpt (2 lines for title max)
- 64x64 image
- Vertically centered “content” block
- Keeping the like/comments metadata
    - I’m doing this because it’s easier to keep it for now and then decide to remove it than remove it now and decide to add it again

cc @develric my latest push already implements these changes.